canonical-websites/www.canonical.com
# Bump global nav version

## Done

- Bump global nav version

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/](http://0.0.0.0:8002/)
4. Make sure navigation has a blog link and not an insights link


## Issue / Card

Fixes [#512](https://github.com/ubuntudesign/web-squad/issues/512)
